awx/main/models/ha.py

```diff
@@ -62,8 +63,11 @@ def has_policy_changes(self):
 
 
 class InstanceLink(BaseModel):
-    source = models.ForeignKey('Instance', on_delete=models.CASCADE, related_name='+')
-    target = models.ForeignKey('Instance', on_delete=models.CASCADE, related_name='reverse_peers')
+    class Meta:
+        ordering = ("id",)
+
+    source = models.ForeignKey('Instance', on_delete=models.CASCADE)
+    target = models.ForeignKey('ReceptorAddress', on_delete=models.CASCADE)
```

Comment:
Please can we get `help_text` added to these? I don't feel like the field names here ever were self-explanatory, but with the model for the target changing, we go far beyond any ability to naturally decipher it. This is particularly explosive because the value is an integer that used to represent the id of one model, and now is the id of a different model, and may be the same, or different, as before the migration, arbitrarily.